locales/ru-RU.json

```diff
@@ -0,0 +1,1003 @@
+        "18": "Едьте в на <CARDINAL_DIRECTION> по <BEGIN_STREET_NAMES>."
+      },
+     "cardinal_directions": ["север", "северо-восток", "восток", "юго-восток", "юг", "юго-запад", "запад", "северо-запад"],
+     "empty_street_name_labels": ["аллее", "велосипедная дорожке", "горной велосипедной тропе"],
+      "metric_lengths": ["<KILOMETERS> километров", "1 километр", "половину километра", "<METERS> метров", "метров"],
+      "us_customary_lengths": ["<MILES> миль", "1 миля", "половину мили", "<TENTHS_OF_MILE> десятых мили", "1 десятую мили", "<FEET> футов", "меньше 10 футов"],
+      "example_phrases": {
+        "0": ["Направляйтесь на восток половину мили.", "Направляйтесь на север один километр."],
+        "1": ["Направляйтесь на юго-восток по Пятой Авеню 1 десятую мили."],
+        "2": ["Направляйтесь на юг по North Prince Street, U.S. 2 22."],
+        "4": ["Едьте на east половину мили.", "Едьте на for 1 километр."],
+        "5": ["Едьте на юго-запад по Пятой Авеню 1 десятую мили."],
+        "6": ["Едьте на юг по North Prince Street, U.S. 2 22."],
+        "8": ["Идите на восток половину мили.", "Идите на север 1 километр."],
+        "9": ["Идите на юго-запад по Пятой Авеню 1 десятую мили."],
+        "10": ["Идите на юг по North Prince Street, U.S. 2 22."],
+        "16": ["Едьте на восток половину мили.", "Едьте на север for 1 километр."],
+        "17": ["Едьте на югозапад on 5th Avenue for 1 десятую мили."],
+        "18": ["Едьте на юг по North Prince Street, U.S. 2 22."]
+      }
+    },
+    "destination": {
+      "phrases": {
+        "0": "Вы прибыли по месту назначения.",
+        "1": "Вы прибыли к <DESTINATION>.",
+        "2": "Местоназначение  <RELATIVE_DIRECTION>.",
+        "3": "<DESTINATION> находится <RELATIVE_DIRECTION>."
+      },
+      "relative_directions": ["слева", "справа"],
+      "example_phrases": {
+        "0": ["Вы прибыли по месту назначения."],
+        "1": ["Вы прибыли к 3206 Powelton Avenue."],
+        "2": ["Местоназначение слева.", "Местоназначение слева."],
+        "3": ["Lancaster Brewing Company находится слева."]
+      }
+    },
+    "destination_verbal_alert": {
+      "phrases": {
+        "0": "Вы прибываете к месту назначения.",
+        "1": "Вы прибываете к <DESTINATION>.",
+        "2": "Место вашего назначения будет <RELATIVE_DIRECTION>.",
+        "3": "<DESTINATION> будет <RELATIVE_DIRECTION>."
+      },
+      "relative_directions": ["слева", "справа"],
+      "example_phrases": {
+        "0": ["Вы прибываете к месту назначения."],
+        "1": ["Вы прибываете к 32 o6 Powelton Avenue."],
+        "2": ["Место вашего назначения будет слева.", "Место вашего назначения будет справа."],
+        "3": ["Lancaster Brewing Company будет справа."]
+      }
+    },
+    "destination_verbal": {
+      "phrases": {
+        "0": ["Вы прибыли по месту назначения."],
+        "1": ["Вы прибыли к <DESTINATION>."],
+        "2": ["Местоназначение слева.", "Местоназначение слева."],
```

Comment:
@krava2008 can you fix this phrase. there should be only one in here and it should have `<RELATIVE_DIRECTION>`